css/selectors/parsing/parse-part.html

```diff
@@ -31,4 +31,7 @@
   test_invalid_selector(":part('foo')");
   test_invalid_selector(":part([foo])");
   test_invalid_selector('::part(foo) + ::part(bar)');
+  test_invalid_selector(":part(foo):is(ul)");
+  test_invalid_selector(":part(foo):is(nav ul)");
+  test_invalid_selector(":part(foo):has(li)");
```

Comment:
You'll want to test the double colon version of part, single colon is always guaranteed to be unsupported.

Reply:
@nt1m Do the tests from line 29 to 32 need to be updated as well or is the single colon syntax part of why those ones are invalid?

Reply:
>  is the single colon syntax part of why those ones are invalid?

This. These are testing that the single-colon syntax (on selectors that would be valid otherwise) are invalid.